cmd/eval.go

```diff
@@ -602,7 +602,12 @@ func setupEval(args []string, params evalCommandParams) (*evalContext, error) {
 		}
 	}
 
-	regoArgs = append(regoArgs, rego.SkipBundleVerification(true), rego.Target(params.target.String()))
+	// skip bundle verification
+	regoArgs = append(regoArgs, rego.SkipBundleVerification(true))
```

Comment:
nit, but if this is unconditional, why not just add this when regoArgs are created above?

Reply:
I don't know! I'll clean it up.